test/prometheus_test.go

```diff
@@ -0,0 +1,139 @@
+package test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/blang/semver"
+	"github.com/google/uuid"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/kind/pkg/apis/config/v1alpha3"
+	"sigs.k8s.io/kind/pkg/cluster"
+
+	"github.com/mesosphere/kubeaddons/pkg/test"
+	"github.com/mesosphere/kubeaddons/pkg/test/cluster/kind"
+	"github.com/mesosphere/kubeaddons/pkg/test/loadable"
+)
+
+func TestPrometheusUp(t *testing.T) {
+	version, err := semver.Parse(defaultKubernetesVersion)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	u := uuid.New()
+
+	node := v1alpha3.Node{}
+	if err := createNodeVolumes(3, u.String(), &node); err != nil {
+		t.Fatal(err)
+	}
+	defer func() {
+		if err := cleanupNodeVolumes(3, u.String(), &node); err != nil {
+			t.Logf("error: %s", err)
+		}
+	}()
+
+	tcluster, err := kind.NewCluster(version, cluster.CreateWithV1Alpha3Config(&v1alpha3.Cluster{Nodes: []v1alpha3.Node{node}}))
```

Comment:
We shouldn't need to deploy a new `test.Cluster` here, but instead utilize the `test.Cluster` already provisioned by the prometheus testing group UNLESS (related to above) we intend to make this the new `TestPrometheusGroup()` function, which is potentially valid.